src/current/cockroachcloud/use-managed-service-backups.md

```diff
@@ -5,42 +5,58 @@ toc: true
 docs_area: manage
 ---
 
-This page describes how to use [managed-service backups](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/backup-and-restore-overview#cockroachdb-backup-types) from CockroachDB {{ site.data.products.serverless }} and CockroachDB {{ site.data.products.dedicated }} clusters.
+This page describes how to use [_managed-service backups_]({% link cockroachcloud/backup-and-restore-overview.md %}) in CockroachDB {{ site.data.products.serverless }} and CockroachDB {{ site.data.products.dedicated }} clusters.
```

Comment:
Only the start of this page before the filters is bulk-new content, a couple of edits here and there after.